packages/runtime/src/index.ts

```diff
@@ -148,12 +150,13 @@ export default function init(inlineConfig: RuntimeOptions = {}) {
       await scheduleUpdate()
   }
 
-  async function extractAll() {
+  async function extractAll(decode?: boolean) {
     const html = document.body && document.body.outerHTML
     if (html) {
-      await extract(html)
-      removeCloak()
+      await extract(decode ? decodeHtml(html) : html)
```

Comment:
Can we just do it in one go without the options? like `extract(html + decodeHtml(html))`?